docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -107,16 +109,18 @@ T6scfmuhWC/TOcm83EVoCzIV3R5dOTKHqkjIUg==
 -----END PGP SIGNATURE-----
 ```
 
-## Troubleshooting issues while signing your plugin
+## Troubleshooting
+
+### Why do I get a "Modified signature" error?
 
-### Why am I getting a "Modified signature" in Grafana?
+Because of an issue when signing the plugin on Windows, in some cases an invalid MANIFEST.txt is generated. You can fix this by replacing all double backslashes, `\\`, with a forward slash, `/` in the MANIFEST.txt file. You need to do this every time you sign your plugin.
```

Comment:
```suggestion
In some cases an invalid `MANIFEST.txt` is generated because of an issue when signing the plugin on Windows. You can fix this by replacing all double backslashes, `\\`, with a forward slash, `/`, in the `MANIFEST.txt` file. You need to do this every time you sign your plugin.
```